website/www/site/i18n/home/hero/en.yaml

```diff
@@ -10,9 +10,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- quote: "A framework that delivers the flexibility and advanced functionality our customers need."
-  name: –Talend
-- quote: "Apache Beam has powerful semantics that solve real-world challenges of stream processing."
-  name: –PayPal
-- quote: "Apache Beam represents a principled approach for analyzing data streams."
-  name: –data Artisans
+- id: home-hero-title
+  translation: Introducing Apache Beam
+- id: home-hero-heading
+  translation: An advanced unified programming model
+- id: home-hero-subheading
+  translation: Implement batch and streaming data processing jobs that run on any execution engine.
+- id: home-hero-button
+  translation: Install Beam
```

Comment:
Is this actually a move? The files seem unrelated

Reply:
This file contains texts of Hero component (the Image Cover on header).

Reply:
Oh, I got ur point. It's weird. The `remove` & `add` actions are in separated commits. Idk why git did that behind the scene. Do you have any exp for this case? 😮

Reply:
Interesting. If they're in separate commits, maybe github is just doing this incorrectly?